roles/network_plugin/weave/templates/weave.j2

```diff
@@ -0,0 +1,6 @@
+WEAVE_PEERS="{% for host in groups['k8s-cluster'] %}{{ hostvars[host]['ip'] | default( hostvars[host]['ansible_default_ipv4']['address']) }}{% if not loop.last %} {% endif %}{% endfor %}"
```

Comment:
My guess is that we should use the access_ip, ip, default pattern here, but haven't tried it yet.  Or wrap into a var like the above flannel_public_ip....